packages/core/strapi/lib/commands/actions/transfer/__tests__/transfer.test.js

```diff
@@ -69,6 +70,7 @@ describe('Transfer', () => {
             onDiagnostic: jest.fn().mockReturnThis(),
           },
           onSchemaDiff: jest.fn(),
+          onAssetsBackupError: jest.fn(),
```

Comment:
Could we make this more generic so that we don't have to create a new listener and type of handler for every error?

I'm mostly concerned that if we add `.onAssetsBackupError(fn)` we'll start accumulating handlers we'll have to deprecate later, versus starting with an `.onError(type, fn)` (or .onResolvableError if we want to be clear)

I actually wanted to do it when I added this for diffs, but since schema diffs weren't quite errors I couldn't fit it in.

Reply:
Sounds good, should I change the `onSchemaDiff` one?

Reply:
If you can make it make sense, but that one seems different to me because it's less "handling an error" and more a series of "handling a subset of an error"